.github/workflows/builds.yml

```diff
@@ -185,6 +185,7 @@ jobs:
                 automake \
                 cargo-vendor \
                 diffutils \
+                dpdk \
```

Comment:
I think some configure should also add `--enable-dpdk`, right ?

Reply:
Good catch, added the switch to CentOS8 build.